src/commands/prepare.ts

```diff
@@ -336,6 +341,8 @@ async function execPublish(remote: string, newVersion: string): Promise<never> {
  * @param changelogPath Path to the changelog file
  */
 async function prepareChangelog(
+  git: SimpleGit,
+  oldVersion: string,
```

Comment:
We should add these to the docstrings. What is `oldVersion` expecting? Is it the raw version number, or the revision of that version? Does it require a tag?

Reply:
Yeah, good catch. Will update 👍🏻